aip/general/0156.md

```diff
@@ -108,8 +112,16 @@ Furthermore, presenting the Singleton resource as a pseudo-collection in such
 methods enables future expansion to a real collection, should a Singleton be
 found lacking.
 
+### Including `plural` definition
+
+While a Singleton is by definition singular, there are certain cases where
+a Singleton resource may appear in a plural form e.g., is the service supports
```

Comment:
is => if

(I'd make a GitHub suggestion, but it always associates the wrong email address)

Reply:
good catch, done.